scripts/build_experimental_archive.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Define build and output directories
+gitTarget=${1:-"master"}
+buildDir=${2:-"./build"}
+
+# Create build directory and navigate into it
+mkdir -p "${buildDir}" && cd "${buildDir}"
+
+# Clone the repository
+git clone -v https://github.com/ooni/probe-cli.git
+
+# Navigate into the repository
+cd probe-cli
+
+git checkout master
+
+git pull origin
+# Checkout the target branch
+git checkout -c "${gitTarget}"
+
+# Build the probe-cli
+make android
```

Comment:
This command has huge requirements in terms of building, but I reckon many of them are probably already satisfied if one is using Android Studio every day. Trying to spell them out, I think they are:

* the correct version of Java being installed (is that Java 17 now?)
* `ANDROID_HOME` being defined
* the `ANDROID_HOME` must contain the correct version of the NDK and the SDK
* the correct version of Go being installed
* the C/C++ compiler being installed

I think most of these preconditions are checked by `make android`, but I also think we should create a GitHub action installing all the required prerequisites, which would serve two purposes:

1. giving us confidence that this method of building works
2. documenting what are the actual pre-requisites

This action does not need to run at every pull request, rather I suppose it's fine to run it only for commits that have already been merged into the `master` branch.